### PR TITLE
WIBEACHES Tests

### PR DESCRIPTION
**Pull request by skaymen**
Covers WIB-173, 174, and 175
https://internal.cida.usgs.gov/jira/browse/WIBEACHES-173
https://internal.cida.usgs.gov/jira/browse/WIBEACHES-174
https://internal.cida.usgs.gov/jira/browse/WIBEACHES-175

### CODE REVIEW

File: src/main/java/gov/usgs/wim/wdnr/domain/SanitaryData.java [hunk 1]
```diff
@@ -1380,12 +1380,12 @@ public void setValidationErrors(final Set<ConstraintViolation<SanitaryData>> inV
         }
     }
 
-    public void addValidatorResult(final ValidatorResult inValidatorResult) {
-        if (null == validationErrors) {
-            validationErrors = new ValidationResults();
-        }
-        validationErrors.addValidatorResult(inValidatorResult);
-    }
+//    public void addValidatorResult(final ValidatorResult inValidatorResult) {
```
Comment (dsteinich) [inline src/main/java/gov/usgs/wim/wdnr/domain/SanitaryData.java]: I not needed, this should be removed.

File: src/main/resources/mybatis/BeachesRawData.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
 <mapper namespace="beachesRawData">
+    <!--count (*) over (partition by 1) total_rows,-->
```
Comment (dsteinich) [inline src/main/resources/mybatis/BeachesRawData.xml]: Please remove this line.

File: src/test/java/gov/usgs/wim/wdnr/Validation/MonitorSiteValidatorForSanitaryDataTest.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    @Test
+    public void isValidTrueTest() {
+        assertTrue(msv.isValid(null, null));
+    }
+
+    @Test
+    public void isValidFalseTest() {
+        given(value.getBeachSeq()).willReturn("2");
+        given(value.getMonitorSiteSeq()).willReturn("79");
+//        given(dao.checkMonitorSite(anyMap())).willReturn((int) 0);
```
Comment (dsteinich) [inline src/test/java/gov/usgs/wim/wdnr/Validation/MonitorSiteValidatorForSanitaryDataTest.java]: Please delete if not needed.

File: src/test/java/gov/usgs/wim/wdnr/domain/SanitaryDataTest.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    @Autowired
+    private Validator validator;
+
+    @Test
+    public void setValidationErrorsTest() {
+
+        SanitaryData sd = new SanitaryData();
+        //TODO: fix this
```
Comment (dsteinich) [inline src/test/java/gov/usgs/wim/wdnr/domain/SanitaryDataTest.java]: Is this TODO still applicable?

File: src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+    @Mock
+    private SanitaryDataDao sanitaryDataDao;
+    @Mock
+    private Validator validator;
+    @Mock
+    private Authentication auth;
+    @Mock
+    private SecurityContext securityContext;
+//    @Mock
```
Comment (dsteinich) [inline src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java]: Please delete if not needed.

File: src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+    private List<SanitaryData> sd;
+//    @Mock
```
Comment (dsteinich) [inline src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java]: Please delete if not needed.

File: src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+    private Set<ConstraintViolation<SanitaryData>> errors;
+//    @Mock
```
Comment (dsteinich) [inline src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java]: Please delete if not needed.

File: src/test/resources/testData/dbCreate.sql [hunk 8]
```diff
@@ -16,3 +16,113 @@ create table monitor_site_dim
 ,update_date                    datetime
 ,status                         varchar(1)
 );
+
```
Comment (dsteinich) [inline src/test/resources/testData/dbCreate.sql]: This entire file is probably not needed now that there is a real Oracle CI database.

File: src/main/java/gov/usgs/wim/wdnr/domain/SanitaryData.java (hunk 1 above)
Comment (skaymen) [reply src/main/java/gov/usgs/wim/wdnr/domain/SanitaryData.java]: done

File: src/main/resources/mybatis/BeachesRawData.xml (hunk 2 above)
Comment (skaymen) [reply src/main/resources/mybatis/BeachesRawData.xml]: done

File: src/test/java/gov/usgs/wim/wdnr/Validation/MonitorSiteValidatorForSanitaryDataTest.java (hunk 3 above)
Comment (skaymen) [reply src/test/java/gov/usgs/wim/wdnr/Validation/MonitorSiteValidatorForSanitaryDataTest.java]: done

File: src/test/java/gov/usgs/wim/wdnr/domain/SanitaryDataTest.java (hunk 4 above)
Comment (skaymen) [reply src/test/java/gov/usgs/wim/wdnr/domain/SanitaryDataTest.java]: no. removed.

File: src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java (hunk 5 above)
Comment (skaymen) [reply src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java]: done

File: src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java (hunk 6 above)
Comment (skaymen) [reply src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java]: done

File: src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java (hunk 7 above)
Comment (skaymen) [reply src/test/java/gov/usgs/wim/wdnr/webservice/SanitaryDataControllerTest.java]: done

File: src/test/resources/testData/dbCreate.sql (hunk 8 above)
Comment (skaymen) [reply src/test/resources/testData/dbCreate.sql]: deleted.

**Pull request merged**